src/v2/Apps/Artwork/Components/SubmittedOrderModal/SubmittedOrderModal.tsx

```diff
@@ -0,0 +1,69 @@
+import { Button, Message, ModalDialog, Spacer, Text } from "@artsy/palette"
+import { FC, useState } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import {
+  getStatusCopy,
+  continueToInboxText,
+} from "v2/Apps/Order/Utils/getStatusCopy"
+import { RouterLink } from "v2/System/Router/RouterLink"
+import { SubmittedOrderModal_me } from "v2/__generated__/SubmittedOrderModal_me.graphql"
+
+interface SubmittedOrderModalProps {
+  me: SubmittedOrderModal_me
+  slug: string
+}
+
+const SubmittedOrderModal: FC<SubmittedOrderModalProps> = ({ slug, me }) => {
+  const submittedOrder = me!.orders!.edges![0]!.node!
+  const submittedOrderArtwork = submittedOrder.lineItems!.edges![0]!.node!
+    .artwork!
+
+  const [isOpen, setIsOpen] = useState(submittedOrderArtwork.slug === slug)
+
+  if (!isOpen) return null
+
+  const handleClose = () => setIsOpen(false)
+  const { title, description } = getStatusCopy({
+    displayState: "SUBMITTED",
+    mode: "OFFER",
+    stateExpiresAt: submittedOrder.stateExpiresAt,
+  })
+
+  return (
+    <ModalDialog title={title as string} onClose={handleClose}>
+      <Message>{description}</Message>
+      <Spacer mb={2} />
+      <Text>{continueToInboxText}</Text>
+      <RouterLink to="/user/conversations" onClick={handleClose}>
+        <Button mt={2} width="100%">
+          Go to Inbox
+        </Button>
+      </RouterLink>
+    </ModalDialog>
+  )
+}
+
+export const SubmittedOrderModalFragmentContainer = createFragmentContainer<
+  SubmittedOrderModalProps
+>(SubmittedOrderModal, {
+  me: graphql`
+    fragment SubmittedOrderModal_me on Me {
+      orders(states: [SUBMITTED], mode: OFFER) {
+        edges {
+          node {
+            stateExpiresAt(format: "MMM D")
+            lineItems {
```

Comment:
Only `stateExpiresAt` is needed on this modal dialog. Added this artwork slug check to be sure the latest order is also the one being displayed on the artwork page.

Reply:
Humm I thought we had something like `myLastOffer` around, sorry to point you in the wrong direction.
It's logical but we can't assure that `const submittedOrder = me!.orders!.edges![0]!.node!` is gonna get exactly the order related to this artwork but I guess we can live with it
@jo-rs thoughts, objections?

Reply:
We do have that but that's related to a specific order object, so I don't think it'd apply here.
Taking a look at `exchange` and `metphysics` really quick. We had this issue at some point with bids where we were pulling the last bid in this way, but it arrived not sorted correctly. We might need to add sorting to the query just in case, and might as well limit it to 1, since as far as I understand we only need one, no?